static/bidder-info/bliink.yaml

```diff
@@ -0,0 +1,21 @@
+maintainer:
+  email: "support@bliink.io"
+gvlVendorID: 658
+modifyingVastXmlAllowed: true
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+      - video
+      - audio
+      - native
+  site:
+    mediaTypes:
+      - banner
+      - video
+      - audio
+      - native
```

Comment:
I saw in your [docs PR](https://github.com/prebid/prebid.github.io/pull/3770/files#diff-895daeea345d41a76783ee2e264598d2c58f17559236624c287aee60772b007c), that only banner and video media types were present, but I see banner, video, audio, and native here. Could you address this difference?

Reply:
Hi, I've fixed mediaTypes and the [documentation](https://github.com/prebid/prebid.github.io/pull/3784)